rasr/feature_scorer.py

```diff
@@ -126,25 +129,54 @@ def __init__(self, prior_mixtures, scale=1.0, priori_scale=0.0, prior_file=None)
         self.config.normalize_mixture_weights = False
 
 
-class TFLabelContextFeatureScorer(FeatureScorer):
+class OnnxFeatureScorer(FeatureScorer):
     def __init__(
         self,
-        fs_tf_config,
-        contextPriorFile,
-        diphonePriorFile,
-        prior_mixtures,
-        prior_scale,
+        *,
+        mixtures: tk.Path,
+        model: tk.Path,
+        io_map: Dict[str, str],
+        label_log_posterior_scale: float = 1.0,
+        label_prior_scale: float = 0.7,
+        label_log_prior_file: tk.Path = None,
```

Comment:
```suggestion
        label_log_prior_file: Optional[tk.Path] = None,
```

I suggest Optional as it can be None. Import Optional from typing

Reply:
done